cmake.config/CMakeLists.txt

```diff
@@ -5,6 +5,26 @@ include(CheckIncludeFiles)
 include(CheckCSourceRuns)
 include(CheckCSourceCompiles)
 
+check_c_source_compiles("
```

Comment:
nice. is it feasible to mention in a comment at the top of cmakelists.txt, something like

> these things (`check_c_source_compiles `, foo, bar, ...) should be done in the *config* phase (cmake.config/), these other things are done in ...)

or is it too situation-dependent ?

Reply:
oh i see this wouldn't make sense for subproject-specific checks (treesitter etc).